bin/deploy-dev.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -ex
+
+CONTAINER_USER="sail"
+COMPOSE_CMD="/usr/local/bin/docker-compose -f docker-compose.cloud.yml"
+EXEC_CMD="$COMPOSE_CMD exec -T --user $CONTAINER_USER app"
+
+# Fetch updates
+git checkout -f
+git checkout dev
+git pull
+
+# Build image and (re)create containers
+$COMPOSE_CMD up -d --force-recreate --build
+
+# Deploy app
+$EXEC_CMD npm install
+$EXEC_CMD composer install --optimize-autoloader
```

Comment:
After assets are built, we should run this to put the site in maintenance mode:

```suggestion
$EXEC_CMD ./artisan down --render="maintenance"
$EXEC_CMD composer install --optimize-autoloader
```